x-pack/plugins/ml/public/application/access_denied/page.tsx

```diff
@@ -30,44 +20,35 @@ export const Page = () => {
   } = useMlKibana();
   const helpLink = docLinks.links.ml.guide;
   return (
-    <Fragment>
+    <>
       <NavigationMenu tabId="access-denied" />
-      <EuiPage>
-        <EuiPageBody>
-          <EuiPageContentHeader>
-            <EuiPageContentHeaderSection>
-              <EuiTitle>
-                <h1>
+      <EuiSpacer size="xxl" />
+      <EuiFlexGroup justifyContent="spaceAround">
+        <EuiFlexItem grow={false}>
+          <EuiPageContent verticalPosition="center" horizontalPosition="center" color="danger">
+            <EuiEmptyPrompt
+              iconType="alert"
+              title={
+                <h2>
                   <FormattedMessage
                     id="xpack.ml.management.jobsList.accessDeniedTitle"
                     defaultMessage="Access denied"
                   />
-                </h1>
-              </EuiTitle>
-            </EuiPageContentHeaderSection>
-          </EuiPageContentHeader>
-          <EuiPageContentBody>
-            <EuiSpacer size="m" />
-            <EuiCallOut
-              title={i18n.translate('xpack.ml.accessDenied.label', {
-                defaultMessage: 'Insufficient permissions',
-              })}
-              color="danger"
-              iconType="cross"
-            >
-              <EuiText size="s">
+                </h2>
+              }
+              body={
                 <p>
                   <FormattedMessage
                     id="xpack.ml.accessDenied.description"
-                    defaultMessage="You don’t have permission to access the ML plugin"
+                    defaultMessage="You don’t have permission to access the Machine Learning plugin."
```

Comment:
> defaultMessage="You don’t have permission to access the Machine Learning plugin."

Is there some tip or more specific details we can provide here to help them resolve the problem? I'm not sure what I need to do get access to the plugin otherwise. e.g. "You must have xxx cluster privileges to ...." or "You must have `all` Kibana privileges for the machine learning feature in this space"?

Reply:
Added a section about the ML feature needing to be visible in the space in https://github.com/elastic/kibana/pull/103117/commits/8caf4d90e9754cf4c83e834f9a799642bbd77c50.